src/components/Button/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Button.css';
+
+interface ButtonProps {
+  handleClick: (e: React.MouseEvent<HTMLButtonElement>) => any,
+  value: string,
+}
+
+const Button: React.FC<ButtonProps> = ({ handleClick, value }) => <button type="button" id="button" onClick={(handleClick)}>{value}</button>;
+
+
+Button.propTypes = {
```

Comment:
Having the interface to type check the props of a component, is useless to double-check them with the prop types. Going for Typescript, you already have all the tools you need for type-checking.

Reply:
React complains if we don't create PropTypes. And typescript complains if we don't create types for it! It seems that we need both... :/